src/redux/categories/categoriesSlice.js

```diff
@@ -9,7 +9,11 @@ const categoriesSlice = createSlice({
   initialState,
   reducers: {
     checkStatus: () => 'Under construction',
+    filterCategory: (state, action) => {
+      state.categories.filter((category) => category === action.payload);
+    },
   },
+
 });
```

Comment:
- [ ] It is required that when the `Check Status` button is clicked, `Under Construction` text should get displayed. At this moment, when I clicked the said button, nothing displays. Kindly look into that and fix it.